source/sass/views/youtube-regrets.scss

```diff
@@ -3,6 +3,170 @@
     background: linear-gradient($youtube-regrets-light-red, $white);
   }
 
+  .intro-viewport {
+    --scenePerspective: 50;
+    --blockZTranslate: 0;
+    --ringZTranslate: 0;
+    --sceneDepth: 0;
+
+    $numRings: 5;
+
+    min-height: 100vh;
+    height: calc(var(--sceneDepth) * 1px);
+
+    .scene-container {
+      position: fixed;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+
+      .scene {
+        position: absolute;
+        left: 0;
+        right: 0;
+        top: 0;
+        bottom: 0;
+        margin: auto;
+        transform-style: preserve-3d;
+        transform: translate3D(0, 0, 0);
+        will-change: transform;
+        perspective-origin: 50% 50%;
+        perspective: calc(var(--scenePerspective) * 1px);
+        width: 100vw;
+        height: 100vw;
+
+        @media (min-width: $bp-sm) {
+          width: 60vw;
+          height: 60vw;
+        }
+
+        .block {
+          position: absolute;
+          display: block;
+          width: 100%;
+          opacity: 0;
+
+          @function get-block-z-tranlsate($block-index) {
```

Comment:
typo, `translate` (all the other ones have the same typo, so it works, but better to fix =D)